LinearAlgebraInOSCAR.ipynb

```diff
@@ -0,0 +1,1003 @@
+       "[4   5   6]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "m3 = matrix(QQ, m2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "9364be30-1897-46d6-bb4d-812d504e8a34",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "QQMatrix"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "typeof(m3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b290602",
+   "metadata": {},
+   "source": [
+    "There are also convenience functions for things like Identity matrices."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "cf02c02e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[1   0   0   0]\n",
+       "[0   1   0   0]\n",
+       "[0   0   1   0]\n",
+       "[0   0   0   1]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Iz = identity_matrix(ZZ, 4)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "89bab643",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[1   0   0   0]\n",
+       "[0   1   0   0]\n",
+       "[0   0   1   0]\n",
+       "[0   0   0   1]"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Iq = identity_matrix(QQ, 4)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "45d05609",
+   "metadata": {},
+   "source": [
+    "If you define a finite field, say $F_9$, you can have an identity matrix with entries in $F_9$ too"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "f61e3acf",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[1   0   0   0]\n",
+       "[0   1   0   0]\n",
+       "[0   0   1   0]\n",
+       "[0   0   0   1]"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "F, a = finite_field(9, \"a\")\n",
+    "If = identity_matrix(F, 4)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f9f7b4c6",
+   "metadata": {},
+   "source": [
+    "We also have a convenience function for zero matrices"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "6c783460",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[0   0   0   0   0]\n",
+       "[0   0   0   0   0]\n",
+       "[0   0   0   0   0]\n",
+       "[0   0   0   0   0]"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "zero_matrix(ZZ, 4, 5)"
```

Comment:
How about ending the section (and perhaps each section in general) with a pointer on where to learn more: 
> Additional matrix constructors such as `diagonal_matrix`, `lower_triangular_matrix`, etc. are described [in the OSCAR manual](https://docs.oscar-system.org/stable/AbstractAlgebra/matrix/#Basic-matrix-functionality).